src/libraries/System.Runtime/tests/System.Runtime.Tests.csproj

```diff
@@ -121,6 +121,7 @@
     <Compile Include="System\PseudoCustomAttributeTests.cs" />
     <Compile Include="System\RangeTests.cs" />
     <Compile Include="System\RankExceptionTests.cs" />
+    <Compile Include="System\Reflection\DefaultBinderTests.cs" />
```

Comment:
FWIW I've been placing new reflection tests into the S.Reflection.Tests repo instead of Runtime, unless there are runtime interactions with native code or differences between Mono and Core for example.